python/paddle/nn/__init__.py

```diff
@@ -19,9 +19,9 @@
 from .layer.container import ParameterList  # noqa: F401
 from ..fluid.dygraph.container import Sequential  # noqa: F401
 
-from .clip import ClipGradByGlobalNorm  # noqa: F401
-from .clip import ClipGradByNorm  # noqa: F401
-from .clip import ClipGradByValue  # noqa: F401
+from paddle.nn.clip import ClipGradByGlobalNorm  # noqa: F401
+from paddle.nn.clip import ClipGradByNorm  # noqa: F401
+from paddle.nn.clip import ClipGradByValue  # noqa: F401
```

Comment:
no need to modify? it is OK to modify this way, but the format is different from others below

Reply:
ok，I will revert it to keep it consistent.